cli/src/pcluster/config/cluster_config.py

```diff
@@ -1829,6 +1870,43 @@ def queue_ami(self):
         else:
             return None
 
+    def get_managed_placement_group_keys(self) -> List[str]:
+        managed_placement_group_keys = []
+        if self.networking.placement_group.is_enabled_and_unassigned:
+            for resource in self.compute_resources:
+                if resource.networking.placement_group.should_be_managed:
+                    managed_placement_group_keys.append(f"{self.name}-{resource.name}")
+        else:
+            for resource in self.compute_resources:
+                if resource.networking.placement_group.is_enabled_and_unassigned:
+                    managed_placement_group_keys.append(f"{self.name}-{resource.name}")
```

Comment:
the logic here is slightly more performant because it only checks if the queue level is enabled once instead of checking it for every compute resource, but I would be OK simplifying if that is more important.  Do you think that is the case here?

Reply:
will simplify the logic here